Nullpointer exception in SpeakerResponseProcessor.java

**Actual Behaviour**
If the speaker's "session" array list is empty, the code throws a null pointer exception and the app crashes on starting. This is because gson is interpreting the empty array list as null and so `speakers.getsession()` is null.

**Expected Behaviour**

There should be a check for empty array list.


**LogCat for the issue**

` Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.util.ArrayList.size()' on a null object reference  at org.fossasia.openevent.api.processor.SpeakerListResponseProcessor.lambda$onResponse$0(SpeakerListResponseProcessor.java:43)`


**Would you like to work on the issue?**

Yes.

**Target Milestone to solve the issue**

Till which [milestone](https://github.com/fossasia/open-event-android/milestones) will you be fixing the issue? 
https://github.com/fossasia/open-event-android/milestone/27

--- a/android/app/src/main/java/org/fossasia/openevent/api/processor/SpeakerListResponseProcessor.java
+++ b/android/app/src/main/java/org/fossasia/openevent/api/processor/SpeakerListResponseProcessor.java
@@ -31,10 +31,12 @@ public class SpeakerListResponseProcessor implements Callback<List<Speaker>> {
                 ArrayList<String> queries = new ArrayList<>();
 
                 for (Speaker speaker : response.body()) {
-                    for (int i = 0; i < speaker.getSession().size(); i++) {
-                        SessionSpeakersMapping sessionSpeakersMapping = new SessionSpeakersMapping(speaker.getSession().get(i).getId(), speaker.getId());
-                        String query_ss = sessionSpeakersMapping.generateSql();
-                        queries.add(query_ss);
+                    if (speaker.getSession() != null) {
+                        for (int i = 0; i < speaker.getSession().size(); i++) {
+                            SessionSpeakersMapping sessionSpeakersMapping = new SessionSpeakersMapping(speaker.getSession().get(i).getId(), speaker.getId());
+                            String query_ss = sessionSpeakersMapping.generateSql();
+                            queries.add(query_ss);
+                        }
                     }
                     String query = speaker.generateSql();
                     queries.add(query);

